doc/admin/MAINTAINER_PROCESS.md

```diff
@@ -0,0 +1,98 @@
+various libseccomp-golang maintainers.  It is not intended as a hard
+requirement, but rather as a guiding document intended to make it easier for
+multiple co-maintainers to manage the libseccomp-golang project.
+
+We recognize this document, like all other parts of the libseccomp-golang
+project, is not perfect.  If changes need to be made, they should be made
+following the guidelines described here.
+
+### Reviewing and Merging Patches
+
+In a perfect world each patch would be independently reviewed and ACK'd by each
+maintainer, but we recognize that is not likely to be practical for each patch.
+Under normal circumstances, each patch should be ACK'd by a simple majority of
+maintainers (in the case of an even number of maintainers, N/2+1) before being
+merged into the repository.  Maintainers should ACK patches using a format
+similar to the Linux Kernel, for example:
+
+```
+Acked-by: John Smith <john.smith@email.org>
+```
+
+The maintainer which merged the patch into the repository should add their
+sign-off after ensuring that it is correct to do so (see the documentation on
+submitting patches); if it is not correct for the maintainer to add their
+sign-off, it is likely the patch should not be merged.  The maintainer should
+add their sign-off using the standard format at the end of the patch's
+metadata, for example:
+
+```
+Signed-off-by: Jane Smith <jane.smith@email.org>
+```
+
+The maintainers are encouraged to communicate with each other for many reasons,
+one of which is to let the others when one is going to be unreachable for an
+extended period of time.  If a patch is being held due to a lack of ACKs and
+the other maintainers are not responding after a reasonable period of time (for
+example, a delay of over two weeks), as long as there are no outstanding NACKs
+the patch can be merged without a simple majority.
+
+### Managing Sensitive Vulnerability Reports
+
+The libseccomp-golang vulnerability reporting process is documented in the
+SECURITY.md document.
+
+The maintainers should work together with the reporter to asses the validity
+and seriousness of the reported vulnerability.
+
+### Managing the GitHub Issue Tracker
+
+We use the GitHub issue tracker to track bugs, feature requests, and sometimes
+unanswered questions.  The conventions here are intended to help distinguish
+between the different uses, and prioritize within those categories.
+
+Feature requests MUST have a "RFE:" prefix added to the issue name and use the
+"enhancement" label.  Bug reports MUST a "BUG:" prefix added to the issue name
+and use the "bug" label.
+
+Issues SHOULD be prioritized using the "priority/high", "priority/medium", and
+"priority/low" labels.  The meaning should hopefully be obvious.
+
+Issues CAN be additionally labeled with the "pending/info", "pending/review",
+and "pending/revision" labels to indicate that additional information is
+needed, the issue/patch is pending review, and/or the patch requires changes.
```

Comment:
I'm not sure that `pending/review` and `pending/revision` labels are needed.

If this is about issues, `pending/review` might make sense (although I was not able to find any issue, either opened of closed, with that label in the seccomp/libseccomp repo). As for `pending/revision`, for issues its meaning is similar to `pending/info`, which essentially asks the reporter to amend the issue in some way. This label is also unused for issues in seccomp/libseccomp.

If this is about patches (pull requests in GH lingo), then `pending/review` is a natural initial state of an opened PR, and IMO does not require a specific label. A reviewer may choose `Request changes` when submitting a review, and GH shows this state, thus having `pending/revision` would be a duplicate of the existing review functionality.

Perhaps this text was written before GH added a review functionality. In any case, I guess it needs to be revised (in seccomp/libseccomp main repo, not here, since this is mostly a copy of text from that repo).

Reply:
> I'm not sure that `pending/review` and `pending/revision` labels are needed.

This why the `pending/*` labels are marked as "CAN" not "MUST" or "SHOULD".  I personally find them helpful but I recognize that not everyone does and as a result I don't want to force them on anyone.  However, they are described here, albeit in very broad terms, for those who do want to make use of them.

> If this is about issues, `pending/review` might make sense (although I was not able to find any issue, either opened of closed, with that label in the seccomp/libseccomp repo). As for `pending/revision`, for issues its meaning is similar to `pending/info`, which essentially asks the reporter to amend the issue in some way. This label is also unused for issues in seccomp/libseccomp.

The line between issues and PRs in GH is a bit fuzzy (although they have been slowly growing a bit more distinct over time); while I generally tend to use these labels only for PRs, they could also be used for issues.  See my comment above about the `pending` labels and "CAN".

It is also worth noting that in many cases I tend to clear these `pending` labels when closing the issue/PR, so please don't read anything into the presence of these labels in a repo.  Also, see my "CAN" comments :)

> If this is about patches (pull requests in GH lingo), then `pending/review` is a natural initial state of an opened PR, and IMO does not require a specific label. A reviewer may choose `Request changes` when submitting a review, and GH shows this state, thus having `pending/revision` would be a duplicate of the existing review functionality.

The `pending` labels are also a nice indicator that someone has looked at the issue/PR and at least done a basic triage of the issue, even if the triage is simply `pending/review`.

Reply:
I don't usually use the `pending` labels, but I recognize they're helpful to others.  I'm fine with whatever direction you choose here.